src/SAWScript/AST.hs

```diff
@@ -241,7 +260,7 @@ data TyCon
   | ContextCon Context
   deriving (Eq, Show)
 
-data Schema = Forall [Name] Type
+data Schema = Forall [(Pos, Name)] Type
```

Comment:
Perhaps we should use `PosPair Name` here rather than `(Pos, Name)`?

Reply:
PosPair seems to be a thing in saw-core's positions, which are not the same types as these positions. Maybe they should be unified.

I could introduce a PosPair here but I'm not convinced it really buys anything...